schema.sql

```diff
@@ -0,0 +1,140 @@
+CREATE DATABASE IF NOT EXISTS `codestroke`;
+USE `codestroke`;
+
+CREATE TABLE IF NOT EXISTS `cases` (
+  `case_id` int NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `first_name` varchar(30) NOT NULL,
+  `last_name` varchar(30) NOT NULL,
+  `dob` date NOT NULL,
+  `address` text NOT NULL,
+  `gender` varchar(1) NOT NULL,
+  `last_well` datetime NOT NULL,
+  `nok` varchar(40) NOT NULL,
+  `nok_phone` varchar(16) NOT NULL,
+  `medicare_no` varchar(12) DEFAULT NULL,
+  `status` tinyint DEFAULT 0
+);
+
+CREATE TABLE IF NOT EXISTS `case_hospitals` (
+  `id` int NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `case_id` int NOT NULL,
+  `hospital_id` int NOT NULL
+);
```

Comment:
I think you're right, the paramedic app just had a hospital picker so I put it in anyway. Actually I'm not sure how this would work then if all the databases are separate for each hospital. 

`id` is unnecessary, was just a remnant from a previous schema. Will remove!